### ISSUE TITLE
Add a custom thumbnail in the feed page

### ISSUE THREAD

**Issue by qthurier** [airbnb/knowledge-repo#51]
Is there a way to do that during the adding step of a knowledge post? The is a feed_image parameter in the code but I don't see how to use it.

**Comment by matthewwardrop** [comment]
Hey @qthurier! We do have a patch against the old internal version of this code which adds support for just that. We're almost finished deprecating our internal repository, after which time all development will be done here. I'll see if I cannot resurrect it soon.

The way this feature currently is implemented is to allow you to specify in the headers of your post which image should be used as the thumbnail of the post. If none is provided, it will use the first image in the post (as currently). Would that work for you?

**Comment by qthurier** [comment]
Yeah that would be great thanks!

**Comment by matthewwardrop** [comment]
Support now complete in master.

**Comment by qthurier** [comment]
Humm could you provide an example of what the header should like then? 

This doesn't work (I still have the default image):

---
title: An unsuccefull attempt to ..
authors:
- Quentin
tags:
- tag0
- tag1
thumbnail: 2
created_at: 2016-11-11
updated_at: 2016-11-11
tldr: This notebook is extracted from the bitbucket repo ...
---

**Comment by matthewwardrop** [comment]
Hi @qthurier,

You should run this header through add script, which will then hard code the actual path into the the thumbnail field. I've tested it locally, and it all seems to work well. Can you run through how you are adding it to the repository?

Also, can you run `knowledge_repo --repo <repo> version`?

**Comment by qthurier** [comment]
I added the header as a raw nbconvert cell at the top of my notebook, then this sequence of actions: add, submit (merge the new branch) and deploy

knowledge_repo --repo ds_knowledge_repo/ --version :
```
Local version: 0.7.2
INFO:knowledge_repo.repositories.gitrepository:Fetching updates to the knowledge repository...
usage: knowledge_repo [--repo REPO] [--knowledge-branch KNOWLEDGE_BRANCH]
                      [--dev] [--debug] [--noupdate] [-h]
                      {init,create,drafts,status,add,submit,push,preview,runserver,deploy,db_upgrade}
                      ...
knowledge_repo: error: unrecognized arguments: --version
```
